backend/utils/const.js

```diff
@@ -78,6 +79,9 @@ const {
 const DATA_URL = null
 const PRINTFUL_URL = 'https://api.printful.com'
 
+// Service that returns a plaintext IP for a GET request
```

Comment:
Service is free and does not have any rate limiting we need to worry about?

Reply:
https://www.ipify.org/

Though, infinite-caching this would probably be a good idea, since it shouldn't change during an instance run.